backend/api/job.proto

```diff
@@ -243,4 +243,9 @@ message Job {
 
   // Input. Whether the job is enabled or not.
   bool enabled = 16;
+
+  // Optional input field. Whether the job should catch up if behind schedule.
+  // If false, the job will only schedule the latest interval if behind schedule.
+  // If true, the job will catch up on each past interval.
+  bool catchup = 17;
```

Comment:
Could you call it backfill?

Reply:
Can you elaborate? I just thought it will be better to use consistent naming from airflow.

Reply:
I saw Airflow uses them interchangeably. https://airflow.apache.org/docs/stable/scheduler.html#backfill-and-catchup

Not a big deal feel free to ignore.

Reply:
Thanks, I'd stay with catchup because the configuration field is called catchup